backend/uclapi/workspaces/urls.py

```diff
@@ -0,0 +1,10 @@
+from django.conf.urls import url
+
+from . import views
+
+urlpatterns = [
+     url(r'^rooms$', views.get_rooms),
+     url(r'^image$', views.get_image),
+     url(r'^sensors/last_updated$', views.get_survey_max_timestamp),

```

Comment:
this url should probably not be snake case?

Reply:
Oh gosh I forgot we had this massive debate :/ We should probably stop [bikeshedding](http://bikeshed.com/) at some point 😆